velox/functions/prestosql/aggregates/CovarianceAggregates.cpp

```diff
@@ -413,10 +413,13 @@ struct RegrSxyResultAccessor {
 
 struct RegrR2ResultAccessor {
   static bool hasResult(const ExtendedRegrAccumulator& accumulator) {
-    return accumulator.m2Y() != 0 && accumulator.m2X() != 0;
+    return accumulator.m2X() != 0;
```

Comment:
Would you update the documentation to clarify these edge cases?

Reply:
Done